view.rb

```diff
@@ -0,0 +1,39 @@
+class View
+  def introduction
+    puts "Welcome to the Inner Critic Flashcard App! We hope you learn something today, but if not, we really don't care!"
+  end
+
+  def user_interface
+    puts "Would you like to go over your flashcards? (yes/no)"
+  end
+
+  def stop_playing
+    puts "Are you going to procastinate and stop playing?"
+    puts "Really?"
+    puts "But are you really really sure?"
+  end
+
+  def question(deck, current_question_index)
+    puts "#{deck[current_question_index].definition}?"
```

Comment:
I think by knowing the "current_question_index" the view knows too much. I would prefer that you ask the Deck for the "current_card" and then pass that to the view.